curve25519-dalek/Cargo.toml

```
@@ -59,7 +59,7 @@ zeroize = { version = "1", default-features = false, optional = true }
 cpufeatures = "0.2.6"
 
 [target.'cfg(curve25519_dalek_backend = "fiat")'.dependencies]
-fiat-crypto = "0.1.19"
+fiat-crypto = "0.2.0"
```

```suggestion
fiat-crypto = "0.2.1"
```

Re: https://github.com/mit-plv/fiat-crypto/pull/1646 - 

Note to myself: We'll need to test no_std with fiat / other backends and I'm sending proper no_std test to fiat as well